app/scripts/views/mixins/password-mixin.js

```diff
@@ -124,6 +135,31 @@ define(function (require, exports, module) {
       }
     },
 
+    handleShowPasswordLabel (el) {
+      const $passwordEl = $(el);
+      if ($passwordEl.siblings(SHOW_PASSWORD_LABEL_SELECTOR).length === 0) {
+        // if no label exists, create one
+        this.createShowPasswordLabel($passwordEl);
+      }
+      // update the visibility of the label
+      this.updateShowPasswordLabelVisibility($passwordEl);
+    },
+
+    createShowPasswordLabel ($passwordEl) {
+      const $showPasswordLabel = $('<label for="show-password">' + SHOW_MESSAGE + '</label>');
```

Comment:
@shane-tomlinson Question: since the checkboxes seem to have different id's [here](https://github.com/mozilla/fxa-content-server/blob/55488e9f3d29f2d5e3f868b95786cadfb59848d0/app/scripts/templates/complete_reset_password.mustache#L69) and [here](https://github.com/mozilla/fxa-content-server/blob/55488e9f3d29f2d5e3f868b95786cadfb59848d0/app/scripts/templates/complete_reset_password.mustache#L59), what is the partial template supposed to look like? Also, how exactly do I render this partial template? The remaining partial templates seem to have their own `view.js` files as well, should that be done here too?